writeups/vn.txt

```diff
@@ -0,0 +1,133 @@
+    Current Practice][12], [Proposed Standard, Internet Standard][13],
+    [Informational, Experimental or Historic][14])? Why is this the proper type
+    of RFC? Do all Datatracker state attributes correctly reflect this intent?
+Proposed Standard.
+
+12. Have reasonable efforts been made to remind all authors of the intellectual 
+    property rights (IPR) disclosure obligations described in [BCP 79][8]?
+    To the best of your knowledge, have all required disclosures been filed?
+    If not, explain why. If yes, summarize any relevant discussion, including
+    links to publicly-available messages when applicable.
+Yes, there are no disclosures to file.
+
+13. Has each author, editor, and contributor shown their willingness to be listed
+    as such? If the total number of authors and editors on the front page is
+    greater than five, please provide a justification.
+Yes.
+
+14. Document any remaining I-D nits in this document. Simply running the [idnits
+    tool][8] is not enough; please review the ["Content Guidelines" on
+    authors.ietf.org][15]. (Also note that the current idnits tool generates
+    some incorrect warnings; a rewrite is underway.)
+None.
+
+15. Should any informative references be normative or vice-versa? See the [IESG
+    Statement on Normative and Informative References][16].
+Not that I can see.
+
+16. List any normative references that are not freely available to anyone. Did
+    the community have sufficient access to review any such normative
+    references?
+None.
+
+17. Are there any normative downward references (see [RFC 3967][9] and
+    [BCP 97][10]) that are not already listed in the [DOWNREF registry][17]?
+    If so, list them.
+None.
+
+18. Are there normative references to documents that are not ready to be
+    submitted to the IESG for publication or are otherwise in an unclear state?
+    If so, what is the plan for their completion?
+None.
+
+19. Will publication of this document change the status of any existing RFCs? If
+    so, does the Datatracker metadata correctly reflect this and are those RFCs
+    listed on the title page, in the abstract, and discussed in the
+    introduction? If not, explain why and point to the part of the document
+    where the relationship of this document to these other RFCs is discussed.
+No.
+
+20. Describe the document shepherd's review of the IANA considerations section,
+    especially with regard to its consistency with the body of the document.
+    Confirm that all aspects of the document requiring IANA assignments are
+    associated with the appropriate reservations in IANA registries. Confirm
+    that any referenced IANA registries have been clearly identified. Confirm
+    that each newly created IANA registry specifies its initial contents,
+    allocations procedures, and a reasonable name (see [RFC 8126][11]).
+The V2 document references the QUIC version registry correctly.
```

Comment:
VN has IANA considerations too, so we should cover that, possibly noting that the intended codepoint is in the 0-63 range, which is Specification Required